File: _BHCVRP_Python_Version/cujae_inf_citi_om/exceptions/RLC_Exception.py
class RLC_Exception(Exception):
    def __init__(self, rlc=None):
        if rlc is not None:
            self.rlc = rlc
            
    def __init__(self, message=None):
        if message is not None:
            super().__init__(message)

    def get_rlc(self):
        return getattr(self, 'rlc', None)

File: BHCVRP_Java_Version/src/cujae/inf/citi/om/exceptions/RlcException.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package cujae.inf.citi.om.exceptions;

/**
 *
 * @author nanda
 */

public class RlcException extends Exception {
    private int rlc;
    
    public RlcException(int rlc){
        this.rlc = rlc;
    
    }
    
    public int getRLC(){
        return rlc;
    }
    
    public RlcException(String message){   
        super(message);
    }
    
}
